kubernetes-with-otelcol-tracing/values.yaml

```diff
@@ -0,0 +1,61 @@
+baseName: buildbarn
+serviceDnsName: svc.mycompany.fake
+namespace: buildbarn
```

Comment:
This is making the assumption that this namespace already exists, we should probably document that or make sure the namespace is created?

Reply:
Or preferably: could we merge this with the existing deployment? That way we won't have to maintain 2 different deployments (we already have often outdated examples with 3, adding a 4th is bound to make things worse in some way 😄)

Reply:
I am very empathetic to the problem of the deployments lagging behind, though I do think there is value in having multiple examples present, since the configs are quite difficult to grok intuitively, at least in my experience.

I'll add a section to create the namespace, and also parameterize it.